src/buttons/RolesEditionButton/components/RolesEditionDialog/RolesEditionDialog.js

```diff
@@ -171,12 +171,17 @@ export const RolesEditionDialog = ({
           </Grid>
           <Grid container>
             {hasNoAdmin && (
-              <Typography data-cy="no-admin-error-message" variant="caption" color="error" paragraph={true}>
+              <Typography
+                data-cy="no-admin-error-message"
+                variant="caption"
+                color="error"
+                sx={{ marginBottom: '16px' }}
```

Comment:
it's better to use theme spacing instead of hard-coded values :slightly_smiling_face:

Reply:
It's a migration script customisation